### PR TITLE
cmd,storage: fix int conversion type errors

### PR DESCRIPTION
**Pull request by srenatus**
Also fixes a panic in `opa oracle find-definition file:12` is file
didn't exit.

### CODE REVIEW

File: storage/path.go [hunk 1]
```diff
@@ -125,7 +125,7 @@ func (p Path) Ref(head *ast.Term) (ref ast.Ref) {
 	ref = make(ast.Ref, len(p)+1)
 	ref[0] = head
 	for i := range p {
-		idx, err := strconv.ParseInt(p[i], 10, 64)
+		idx, err := strconv.ParseInt(p[i], 10, 32)
```
Comment (tsandall) [inline storage/path.go:128]: Alternatively we could `strconv.ParseUInt` and then call `ast.UIntNumberTerm` below. That way we can index into arrays larger than 2**32? Not sure

**Review by tsandall — APPROVED** [review]
LGTM

File: storage/path.go (hunk 1 above)
Comment (srenatus) [reply storage/path.go:128]: Hmm good idea. I'll try that.

Comment (srenatus) [reply storage/path.go:128]: ....in a follow-up 😎

**Pull request merged**